hypothesis_bio/hypothesis_bio.py

```diff
@@ -38,6 +38,27 @@ def dna(
     return draw(text(alphabet=chars, min_size=min_size, max_size=max_size))
 
 
+@composite
+def fastq_quality(draw, size=0):
+    """Generates the quality string for the FASTQ format
+
+    Arguments:
+    - `size`: Size of the quality string to be generated
+
+    Note:
+    According to https://en.wikipedia.org/wiki/FASTQ_format,
+    the range of characters for the quality string ranges from
+    byte value 0x21 to 0x7E.
+    """
+    return draw(
+        text(
+            alphabet=characters(min_codepoint=33, max_codepoint=121),
```

Comment:
0x7E is 126 but `max_codepoint=121`. Is there a reason for this?

Reply:
Hmm I think I screwed up with my math I think.